packages/forms-web-app/src/dynamic-forms/dynamic-components/list-add-more/question.js

```diff
@@ -64,11 +75,7 @@ class ListAddMoreQuestion extends Question {
 			validators
 		});
 
-		if (!subQuestion || !(subQuestion instanceof AddMoreQuestion)) {
-			throw new Error('addMore is mandatory');
-		}
```

Comment:
This gets caught by the type system. I'm hoping to get questions.js to zero type issues soon